lib/ramble/ramble/spack_runner.py

```diff
@@ -602,20 +602,17 @@ def inventory_hash(self, require_exist=False):
         contents_to_hash = None
 
         if not self.dry_run:
-            if not lock_exists and require_exist and len(self.env_contents) > 0:
-                logger.die(
-                    'spack.lock file does not exist in environment '
-                    f'{self.env_path}\n'
-                    'Make sure your workspace is fully setup with\n'
-                    '    ramble workspace setup'
-                )
-            elif len(self.env_contents) == 0:
-                contents_to_hash = {}
-            elif lock_exists:
+            if not lock_exists:
+                if require_exist and len(self.env_contents) > 0:
```

Comment:
This LGTM

Reply:
As discussed offline I will also add a `status=SETUP` to have a more explicit semantic around if something is setup